### ISSUE TITLE
Cannot open all files created from previous version[Bug]: 

### ISSUE THREAD

**Issue by LoriSap** [jasp-stats/jasp-issues#2458]
### JASP Version

0.18.2

### JASP Module

Quality Control

### What analysis are you seeing the problem on?

Process Capability

### What OS are you seeing the problem on?

Windows 11

### Bug Description

Error
JASp has experienced an unexpected internal error:
The Engine Crashed...
JASP cannot continue and will close

### Expected Behaviour

Program should open and perform the capability analysis

### Steps to Reproduce

1.Yes
it is repeatable on same file, but does not happen on all files
2.

3.
...

### Log (if any)

Sorry ITAR regulations prohibit me from sharing data

### Final Checklist

- [ ] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by LoriSap** [comment]
No, as I mentioned in initial report, my information is regulated by ITAR as it is aerospace and dept. of defense.  I do find it odd where certain files will open, others do not.  And this is not dependent on original version that these files were created.  It is consistent though .  Files that do not open just will not open.

Lori Saponaro
Quality Specialist
SKF USA Inc.
20 Industrial Dr.
Hanover, PA.17331
(717)347-5258
***@***.***

**Comment by LoriSap** [comment]
I do not know what versions this has been failing on or which are working.  As I stated previously, I have checked different files that were created before 17.1, those from that last version, last week and even yesterday.  I really can't tell you why some work and some don't.  Those that don't, will not even open.  Below is the message I get.  And as I said, not all files created do this.  Not eve all files from same date.
***@***.***






Lori Saponaro
Quality Specialist
SKF USA Inc.
20 Industrial Dr.
Hanover, PA.17331
(717)347-5258
***@***.***

**Comment by LoriSap** [comment]
Ok, This seems to work to some extent.  It appears I can open up files from this update and at least last two, still having to reanalyze data from 2022, but that really is not a big issue as long as current files run.
Thank you!

Lori Saponaro
Quality Specialist
SKF USA Inc.
20 Industrial Dr.
Hanover, PA.17331
(717)347-5258
***@***.***

**Comment by JTPetter** [comment]
@LoriSap Great, thanks for trying it and reporting back. The new version with the fixes should be released very soon!

**Comment by LoriSap** [comment]
Good Morning!
I have discovered a glitch to the beta version 18.2 of JASP.  I was so happy that I could create files, I never checked if I could reopen new files.  Whenever I create a new file from scratch, or copy and paste from old file, it will not reload after it is saved.  It will do the analysis, but after closing, I cannot reopen it.  I did report this bug.  Just FYI to you


Lori Saponaro
Quality Specialist
SKF USA Inc.
20 Industrial Dr.
Hanover, PA.17331
(717)347-5258
***@***.***
